### ISSUE TITLE
Quarto not working with sparkline::sparkline

### ISSUE THREAD

**Issue by harrelfe** [quarto-dev/quarto-cli#5717]
### Bug description

`Quarto` version 1.4.90 unlike recent previous versions is not working with `sparkline`.  

### Steps to reproduce

````qmd
---
title: "Test sparklines"
format: html
---

```{r}
sparkline::sparkline(0)   # load sparkline javascript dependencies
```
````

### Expected behavior

Should load `jQuery` dependencies to set up for sparklines, with no visible output.

### Actual behavior

````
Quitting from lines 7-8 [unnamed-chunk-1] (spark.qmd)
Error in `add_html_caption()`:
! unused argument (xfun::grep_sub("^[^<]*<[^>]+aria-labelledby[ ]*=[ ]*\"([^\"]+)\".*$", "\\1", x))
Backtrace:
  1. global .main()
  2. execute(...)
  3. rmarkdown::render(...)
  4. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  5. knitr:::process_file(text, output)
     ...
 14. sew(res, options)
 15. knitr:::sew.list(x, options, ...)
 16. base::lapply(x, sew, options, ...)
 17. FUN(X[[i]], ...)
 18. knitr:::sew.knit_asis(x, options, ...)
Execution halted
````

### Your environment

- OS: Ubuntu 22.04 popOS
- knitr 1.43
- R 4.3.0

### Quarto check output

```bash
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.2: OK
      Dart Sass version 1.55.0: OK
      Deno version 1.33.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.90
      Path: /opt/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.6
      Path: /usr/bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

[✓] Checking R installation...........OK
      Version: 4.3.0
      Path: /usr/lib/R
      LibPaths:
        - /home/harrelfe/R/x86_64-pc-linux-gnu-library/4.3
        - /usr/local/lib/R/site-library
        - /usr/lib/R/site-library
        - /usr/lib/R/library
      knitr: 1.43
      rmarkdown: 2.21

[✓] Checking Knitr engine render......OK
```

**Comment by mcanouil** [comment]
Seems to be related to #5052 and/or #5702.

**Comment by mcanouil** [comment]
I checked, this is a `knitr` 1.43 issue.

**Comment by mcanouil** [comment]
@harrelfe Could You try the latest pre-release version, it should be fixed in it?

**Comment by harrelfe** [comment]
Fixed.  That was fast!

**Comment by mcanouil** [comment]
It was actually fixed yesterday in #5704
